### ISSUE TITLE
Cannot store numeric LocalStorage data

### ISSUE THREAD

**Issue by cpetrov** [eclipsesource/tabris-js#505]
```javascript
localStorage.setItem("patchTimestamp", 1435661735);
console.log(localStorage.getItem("patchTimestamp")); // logs "null" on Android, "undefined" on iOS
```

Stringifying the number value before setting it works as expected.

**Comment by cpetrov** [comment]
This issue was actually related to storing numeric data, so renamed the issue and  updated the description.

**Comment by cpetrov** [comment]
A fix has been merged and will be available in Tabris 1.1.0.
